Makefile

```diff
@@ -604,6 +605,27 @@ $(SBOM): $(ROOTFS_TAR) | $(INSTALLER)
 	rm -rf $(TMP_ROOTDIR)
 	$(QUIET): $@: Succeeded
 
+generate_metadata: $(ROOTFS_TAR) | $(INSTALLER)
```

Comment:
Did you mean this target to be `generate_metadata:` as well, or `$(GENERATE_METADATA)`?

Reply:
It should be `$(GENERATE_METADATA)`. Fixed it